### PR TITLE
Feature/416704 bulkupload n save todb AB#416704

### PR DESCRIPTION
**Pull request by abidkayanidefra**
Feature/416704 bulkupload n save todb

### CODE REVIEW

File: src/EPR.SubsidiaryBulkUpload.Application/Models/Accounts.cs
Comment (BrianKnightEviden) [inline src/EPR.SubsidiaryBulkUpload.Application/Models/Accounts.cs:1]: This shouldn't be here. This is not an accounts service

Comment (mikeymonster) [reply src/EPR.SubsidiaryBulkUpload.Application/Models/Accounts.cs:1]: This is used in CompaniesHouseResponse - I agree that we should be able to remove it, but we can do that in the new branch.

File: src/EPR.SubsidiaryBulkUpload.Application/Services/CompaniesHouseDataProvider.cs [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        var response = await subsidiaryService.GetCompanyByOrgIdFromTableStorage(subsidiaryModel.CompaniesHouseNumber);
+
+        if (response != null)
+        {
+            dataRetrieved = true;
+            subsidiaryModel.Address = response.Address;
+        }
+        else
+        {
+            // Try get remotely through CH API
```
Comment (RamakrishnaGujjula) [inline src/EPR.SubsidiaryBulkUpload.Application/Services/CompaniesHouseDataProvider.cs:25]: Not including any commented lines?

**Review by RamakrishnaGujjula — APPROVED** [review]

File: src/EPR.SubsidiaryBulkUpload.Application/Services/CompaniesHouseDataProvider.cs (hunk 1 above)
Comment (abidkayanidefra) [reply src/EPR.SubsidiaryBulkUpload.Application/Services/CompaniesHouseDataProvider.cs:25]: there are few more comments in other classes. there are really helpful for readability.

**Pull request merged**